docs/maps/import-geospatial-data.asciidoc

```diff
@@ -98,10 +98,6 @@ To open an existing index for drawing:
 
 . Set *Scaling* to *Limit results to 10,000*.
```

Comment:
Can you also remove the line "Set *Scaling* to *Limit results to 10,000*.". This is no longer required.